<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sysSettingDlg</class>
 <widget class="QDialog" name="sysSettingDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>260</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>260</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>系统设置</string>
  </property>
  <widget class="QPushButton" name="comfirmButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>71</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>331</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="serverPortNameLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>服务端口</string>
    </property>
   </widget>
   <widget class="QLabel" name="serverAddrNameLable">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>服务地址</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="autoStartCheckBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>开机时自动启动客户端</string>
    </property>
   </widget>
   <widget class="QLabel" name="autoUpIntervalLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>自动更新时间间隔</string>
    </property>
   </widget>
   <widget class="QLabel" name="hourLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>63</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>时</string>
    </property>
   </widget>
   <widget class="QLabel" name="serverAddrContentLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="serverPortContentLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>13526</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="timelineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>71</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
